learn/tanssi/external-security-providers/symbiotic.md

```diff
@@ -0,0 +1,101 @@
+        developers-- Launch --> networks
+    end
+```
+
+Around these components, different actors participate:
+
+- **Restakers** - are the ones providing liquidity to the vaults, obtaining rewards. Restakers choose which vaults they restake to, provided that the vault accepts their assets and they agree with the networks the vault works with and the general setup
+- **Vault Curators** - are responsible for the vault administration. Curators decide which networks the vault works with, which operators are whitelisted to participate in securing the networks, which assets are accepted as collaterals and the slashing and rewarding mechanisms
+- **Resolvers** - are responsible for resolving veto-slashing events in the vault. Veto slashing events are a particular type of event that requires the participation of resolvers, who have the authority to revoke the slashing request
+- **Node Operators** - are the ones running the actual operators, which validate the networks' transactions. Node operators are responsible for the configuration and hardware of the nodes. They apply to offer their services in both vaults and networks and have to be accepted and whitelisted by both before starting to validate networks and get rewards
+- **Developers** - are the ones building appchains
+
+### Tanssi with Symbiotic {: #tanssi-symbiotic }
+
+Developers launching appchains through Tanssi benefit from [block production services](/learn/tanssi/appchain-services/block-production/){target=\_blank} and the shared security model derived from every vault opting-in to support the Tanssi protocol. This eliminates the hurdle of dealing with infrastructural components developers would need to take on otherwise.
+
+Curators running vaults can apply to offer the restaked collaterals as economic security for the Tanssi network, regardless of how many appchains are running through the Tanssi protocol.
+
+Operators in the vault run the same setup to provide block production and validation services to the Tanssi network and, consequently, to every appchain deployed through Tanssi. This unique architecture facilitates all the tasks related to running and maintaining the operators since there is only one setup.
+
+ All things combined shape a functional and elegant ecosystem where developers can focus on creating and innovating. Tanssi handles the infrastructural components, guaranteeing liveness and performance, and Symbiotic provides the economic safeguards to ensure the validity of the operations.
+
+
+```mermaid
+flowchart LR
+
+Symbiotic  -- Secures --> tanssi["Tanssi Network"]
+tanssi -- Provides Services and
+     Security --> Appchains
+```
+
+### Slashing and Rewards {: #slashing-rewards }
+
+Well-behaved operators and restakers receive rewards for their participation in TANSSI tokens. Reward payments are managed through the vault.
+
+The Tanssi protocol also penalizes bad actors' misbehavings. These are the actions that cause slashing events:
```

Comment:
Be a bit more specific. Tanssi protocol implements a middleware in Symbitoic that handles slashing of operators that are misbehaving. Go through what we've discussed in the past, rogue AVSs are not slashed because what we incentivize is block production and validation. Similar to having a rogue contract in Ethereum does not affect ethereum operation itself, or Ethereum validators are not slashed.